base/multi.jl

```diff
@@ -666,7 +760,36 @@ end
 hash(r::AbstractRemoteRef, h::UInt) = hash(r.whence, hash(r.id, h))
 ==(r::AbstractRemoteRef, s::AbstractRemoteRef) = (r.whence==s.whence && r.id==s.id)
 
+"""
+    Base.remoteref_id(r::AbstractRemoteRef) -> RRID
```

Comment:
They were added in https://github.com/JuliaLang/julia/pull/14053 . Useful when serializing/deserializing only identities of remote references across workers. Both shared arrays and distributed arrays use these functions to avoid the overhead of our distributed gc in certain scenarios,